src/test/java/Homework16.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+@Test
+
+public void registrationNavigation() { {
+
+    ChromeOptions options = new ChromeOptions();
+    options.addArguments("--remote-allow-origins=*");
+    options.addArguments("--start-maximized");
+
+    WebDriver driver = new ChromeDriver(options);
+    driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+    String url = "https://testpro.io/";
+    driver.get(url);
+
+    WebElement registrationLink = driver.findElement(By.cssSelector("//a[@href='registration']"));
+    registrationLink.click();
```

Comment:
// comment from hw team: 
Hi friend,
I have checked your homework 16, your test script run failed because `WebElement`, `By `cannot resolve.

Java comes with a core set of libraries, including those that define commonly used data types and related behavior. Selenium Web Driver encapsulates a simple form element as an object of WebElement. => The first, you need to import the Selenium libraries needed to run the test.
//HINT: `import org.openqa.selenium.By;`
            ` import org.openqa.selenium.WebElement;`